### DISCUSSION TITLE
Error loading dashboard

### DISCUSSION THREAD

**Discussion by Mnstanchev** [vercel/community#7320] [Help] [answered]
### Summary

Guys can someone help, because Vercel isn't. Can someone tell me what are you doing when you try to go to your Dashboard to manage your projects and it appears a big red square with " Error loading dashboard" ?

### Steps to Reproduce

Going into your account --> Dashboard --> clicking on your project --> Big Red Suare with "Error loading dashboard"

**Comment by marciamoss** [comment]
same issue for me.

**Comment by qbdumont** [comment]
Same

**Comment by MEF-Communication** [comment]
same

**Comment by swarnava** [comment]
Hi, 

Swarnava from Vercel Support here. We are actively investigating this issue right now!

Statuspage: https://www.vercel-status.com/incidents/7svxyp760r48

**Comment by swarnava** [comment, marked as answer]
This should be fixed now!

[a comment by an automated account is not shown]
